example/src/main/java/com/fuzzproductions/ratingbarexample/MainActivity.java

```diff
@@ -37,16 +38,27 @@ public void onRatingChanged(RatingBar ratingBar, float rating, boolean fromUser)
                     }
                 }
             });
+            // Should move this elsewhere but below checks that making bar indicator allows
+            // touch events to pass through to parent
```

Comment:
@raigex Address the comment to the reader. Say something like `If you want to see for yourself whether click events are registered correctly, try out the below lines of code.`